src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -233,6 +233,14 @@ namespace AirflowNetworkBalanceManager {
     // Report variables
 
     // MODULE VARIABLE DECLARATIONS:
+
+     namespace {
+        // These are purposefully not in the header file as an extern variable. No one outside of this should
+        // use these. They are cleared by clear_state() for use by unit tests, but normal simulations should be unaffected.
+        // This is purposefully in an anonymous namespace so nothing outside this implementation file can use it.
+        bool ValidateDistributionSystemFlag(true);
```

Comment:
As I indicated on the previous PR, this is NOT the preferred solution and is only exacerbating the problem.

Reply:
So I do agree that adding to the global state here is not the best solution, and it would be better to just ensure we only execute the one-time code...well...once.  However, I will say that pulling out static variables from inside functions is a very important incremental step.  I'm not as opposed to this as a band-aid, and I am now encountering this in my testing and want this to be fixed.  In future fixes, we'll push for more formal solutions, but I'm going to merge this in.